core/gui/src/app/workspace/service/workflow-graph/model/sync-texera-model.spec.ts

```diff
@@ -351,17 +340,7 @@ describe("SyncTexeraModel", () => {
 
       // construct the texera sync model with spied dependencies
```

Comment:
seems to be duplicated comment, we can clean it here

Reply:
Removed.